app/controllers/test_controller.rb

```diff
@@ -158,7 +158,7 @@ def new_bot
 
   def archive_project
     p = Project.find(params[:project_id])
-    p.archived = true
+    p.archived = 1
```

Comment:
So you changed `archived` from `Team` and `Project` to be integers too?

Reply:
yes and for `Source` too